LBHackney-IT/lbh-social-care
# added isMultiInit to define if to show the first input

**What**  

Extend #158. Now `isMulti` can show the related input after clicking on its trigger.

```
        {
          component: 'TextInput',
          name: 'title_2',
          width: '30',
          label: 'Title',
          isMulti: true,
          isMultiInit: false,
          isMultiTrigger: 'Add a title',
        },
```

<img width="790" alt="Screenshot 2021-01-29 at 16 07 30" src="https://user-images.githubusercontent.com/435399/106298712-8569a400-6254-11eb-8c65-0672e1b6195a.png">


**How to test it**
- check create-person form
or
- go to http://dev.hackney.gov.uk:3000/form/test/first-step
- select show object step
- check that is working properly
